server/database/models/User.js

```diff
@@ -14,10 +22,37 @@ const userSchema = mongoose.Schema({
       message: 'User already exists!'
     }
   },
-  password: String,
-  opportunitiesAppliedFor: [mongoose.Schema.Types.ObjectId]
-})
+  password: {
+    type: String,
+    required: true
+  },
+  opportunitiesAppliedFor: [mongoose.Schema.Types.ObjectId],
+  organization: mongoose.Schema.Types.ObjectId
+}, { timestamps: true })
+
+UserSchema.methods.generateJWT = function () {
```

Comment:
Hmm this looks good! I wonder if we could rename `jwt` to `sessionToken` instead for readability and future-proofing. For example for this method, `generateSessionToken`.